EIPS/eip-5791.md

```diff
@@ -0,0 +1,166 @@
+    /// @param blockNumberUsedInSig The block number linked to the blockhash signed in signatureFromChip. Should be a recent block number.
+    /// @param useSafeTransferFrom Whether EIP-721's safeTransferFrom should be used in the implementation, instead of transferFrom.
+    ///
+    /// @dev The implementation should check that block number be reasonably recent to avoid replay attacks of stale signatures.
+    /// The implementation should also verify that the address signed in the signature matches msgSender.
+    /// If the address recovered from the signature matches a chip address that's bound to an existing token, the token should be transferred to msgSender.
+    /// If there is no existing token linked to the chip, the function should error.
+    function transferTokenWithChip(
+        bytes calldata signatureFromChip,
+        uint256 blockNumberUsedInSig,
+        bool useSafeTransferFrom
+    ) external;
+
+    /// @notice Calls transferTokenWithChip as defined above, with useSafeTransferFrom set to false.
+    function transferTokenWithChip(bytes calldata signatureFromChip, uint256 blockNumberUsedInSig) external;
+
+    /// @notice Emitted when a token is minted
+    event PBTMint(uint256 indexed tokenId, address indexed chipAddress);
+
+    /// @notice Emitted when a token is mapped to a different chip.
+    /// Chip replacements may be useful in certain scenarios (e.g. chip defect).
+    event PBTChipRemapping(uint256 indexed tokenId, address indexed oldChipAddress, address indexed newChipAddress);
+}
+
+```
+
+To aid recognition that an [EIP-721](./eip-721.md) token implements physical binding via this EIP: upon calling [EIP-721](./eip-721.md)’s `function supportsInterface(bytes4 interfaceID) external view returns (bool)` with `interfaceID=0xa70d2657`, a contract implementing this EIP must return true.
+
+The mint interface is up to the implementation. The minted NFT's owner should be the owner of the physical chip (this authentication could be implemented using the signature scheme defined for `transferTokenWithChip`).
+
+## Rationale
+
+The interface includes a `transferTokenWithChip` function that's opinionated with respect to the signature scheme, in order to enable a downstream aggregator-like product that supports transfers of any NFTs that implement this EIP in the future.
+
+### Out of Scope
+
+The following are some peripheral problems that are intentionally not within the scope of this EIP:
+
+- trusting that a specific NFT collection's chip addresses actually map to physical chips embedded in items, instead of arbitrary EOAs
+- ensuring that the chip does not deterioriate or get damaged
+- ensuring that the chip stays attached to the physical item
+- etc.
+
+Work is being done on these challenges in parallel. For example, the Ethereum Reality Service (`https://www.ers.to/`) is meant to tackle some of these issues.
```

Comment:
```suggestion
Work is being done on these challenges in parallel.
```

Reply:
Was also wondering what the downsides were of including this link. It's a relatively light landing page that mostly redirects to https://ethereum-magicians.org/t/a-proposal-for-the-ethereum-reality-service/9694. Would a link to ethereum-magicians directly be more appropriate?